scripts/eslint-pre-commit

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-git stash -q --keep-index
-
-# Test prospective commit
-git diff-index --cached HEAD --name-only --diff-filter ACMR | egrep '.js$' | xargs -I $(npm bin)/eslint {}
```

Comment:
* Does that run even when there are no changes js files? Gnu `xargs` (read: Linux) supports `-0`/`--null` to prevent running with no args. I'm not sure what you do for macs though.
* I though `-I` in xargs was when you wanted to drop the files in a certain place. Normal usage is `xargs -I % foo/%/bar`. Since it's at the end, I don't think it's needed.